### PR TITLE
Fix typos in aggregate report

### PR DESCRIPTION
**Pull request by madelondohmen**
### Changes
* Added "server" to the server names (eg. "Web server" instead of "Web")
* Added a title to the vulnerability data to show only the IP-address/hostname in the tabel of contents (like in the design)

### Demo
afbeelding

afbeelding

---

### Code Checklist
- [x] All the commits in this PR are properly PGP-signed and verified;
- [x] This PR only contains functionality relevant to the issue; tickets have been created for newly discovered issues.
- [ ] I have written unit tests for the changes or fixes I made.
- [x] For any non-trivial functionality, I have added integration and/or end-to-end tests.
- [x] I have performed a self-review of my code and refactored it to the best of my abilities.

### Communication
- [x] I have informed others of any required `.env` changes files if required and changed the `.env-dist` accordingly.
- [x] I have made corresponding changes to the documentation, if necessary.

---
## Checklist for code reviewers:
Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---
## Checklist for QA:
Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.

### CODE REVIEW

**Review by ammar92 — DISMISSED** [review]

File: rocky/reports/report_types/aggregate_organisation_report/report.py [hunk 1]
```diff
@@ -99,6 +99,7 @@ def post_process_data(self, data):
                         total_findings += vulnerabilities_data["summary"]["total_findings"]
                         terms.extend(vulnerabilities_data["summary"]["terms"])
                         recommendations.extend(vulnerabilities_data["summary"]["recommendations"])
+                        vulnerabilities_data["title"] = "'" + ip.split("|")[2] + "'"
```
Comment (dekkers) [inline rocky/reports/report_types/aggregate_organisation_report/report.py:102]: Why put single quotes around the IP address? It looks a bit strange to me.

Comment (Donnype) [reply rocky/reports/report_types/aggregate_organisation_report/report.py:102]: Perhaps no quotes ánd **bold** would be even better? Just throwing out an idea

Comment (underdarknl) [reply rocky/reports/report_types/aggregate_organisation_report/report.py:102]: the quotes would be to differentiate from the list numbering I'd say. However, maybe the list numbering should be a different size / style / font from the contents for all these items @HeleenSG ?

Comment (HeleenSG) [reply rocky/reports/report_types/aggregate_organisation_report/report.py:102]: I think we can remove the single quotes. And slightly increase the space between the list numbering and the content. I believe that will be enough. 

@Donnype as the title is already bold, we can't use that to differentiate.

Comment (HeleenSG) [reply rocky/reports/report_types/aggregate_organisation_report/report.py:102]: New design after talking to @underdarknl  and realising this question was mainly about the ip-addresses within the table of content.

Suggested solution is to use italic text for object names (incl ip-addresses) within the toc. E.g see 6.1, 6.2 in the following design:

Screenshot 2023-12-28 at 09 32 32

**Review by ammar92 — DISMISSED** [review]

**Review by underdarknl — APPROVED** [review]

**Pull request merged**